src/panels/lovelace/hui-root.ts

```diff
@@ -497,6 +498,12 @@ class HUIRoot extends LitElement {
         | undefined;
 
       if (!oldLovelace || oldLovelace.config !== this.lovelace!.config) {
+        if (this.lovelace!.config.resources) {
```

Comment:
We can't just restore this, we only want to load resources from the default Lovelace config, not from the other dashboards.